keps/sig-node/1287-in-place-update-pod-resources/kep.yaml

```diff
@@ -32,7 +32,7 @@ replaces:
 
 stage: "alpha"
 
-latest-milestone: "v1.28"
+latest-milestone: "v1.29"
 
 milestone:
   alpha: "v1.27"
```

Comment:
Please update beta milestone to 1.30 and stable to 1.31 for now.
Besides that LGTMIA

Reply:
Sounds good. updated